universal-application-tool-0.0.1/app/auth/saml/SamlCiviFormProfileAdapter.java

```diff
@@ -163,6 +162,15 @@ public CiviFormProfileData mergeCiviFormProfile(
     return civiFormProfile.getProfileData();
   }
 
+  private String extractAttributeFromArrayList(SAML2Profile profile, String attr) {
+    ArrayList attributeArray = profile.getAttribute(attr, ArrayList.class);
+    StringBuilder sb = new StringBuilder();
+    for (Object s : attributeArray) {
+      sb.append(s);
```

Comment:
Since we're supporting a person having multiple names, we should likely separate them with a space.  StringJoiner(" ") will help with that instead of StringBuilder.

https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

Reply:
done

Reply:
As a person with multiple last names, I went to test this, and found that the way we handle names is kind of error prone. First, if there's a space in the first name, it isn't returned as an array list by LoginRadius, just a string with a space in it. Also, if someone has a middle name and two last names, their name is displayed as <<Anonymous Applicant>> which feels wrong. Here's how my name is stored in the DB: 
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/19631367/156236263-246e065c-d302-4c26-94fa-bfdd8eb64a3e.png">

Reply:
filed https://github.com/seattle-uat/civiform/issues/2000 to address this